jest.preset.js

```diff
@@ -17,7 +17,7 @@ const baseConfig = {
     '^.+\\.ts$': 'ts-jest',
   },
   testMatch: ['**/+(*.)+(spec|test).+(ts|js)?(x)'],
-  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx'],
+  moduleFileExtensions: ['ts', 'tsx', 'js', 'jsx', 'json'],
```

Comment:
hmm wondering if this will be really needed for all packages ? maybe I'd override it solely for babel-make-styles ? WDYT?

Reply:
Makes sense, applied it in `packages/babel-make-styles/jest.config.js`